internal/overlord/servstate/handlers.go

```diff
@@ -54,9 +54,15 @@ func TaskServiceRequest(task *state.Task) (*ServiceRequest, error) {
 }
 
 var (
-	okayWait = 1 * time.Second
-	killWait = 5 * time.Second
-	failWait = 10 * time.Second
+	// defaultKillDelay is the duration afforded to services for processing
+	// SIGTERM signals and shutting down cleanly if the service hasn't specified
+	// their own duration.
+	defaultKillDelay = 25 * time.Second
+	okayWait         = 1 * time.Second
```

Comment:
Because `okayWait` is quite separate (and used when starting a service), can we move it to the top of this block? And perhaps add a brief comment while we're here:

```go
var (
    // okayWait is the time to wait after starting a service before we conclude
    // that it's running successfully.
    okayWait = 1 * time.Second
    ...
)
```